toolkit/scripts/toolchain/check_manifests.sh

```diff
@@ -36,8 +36,16 @@ write_rpms_from_toolchain () {
     specs=$(cat $TOOLCHAIN_SPEC_LIST_FILE)
     for specName in $specs
     do
-        specFile=$(find $SPECS_DIR/**/$specName.spec)
-        write_rpms_from_spec $specFile $1
+        if [[ $specName == *"msopenjdk"* ]]; then
+            # special case to add msopenjdk-11 which is downloaded and does not have a SPEC
+            jdkfilename=$(cat "$MANIFESTS_DIR/toolchain_$ARCH.txt" | grep $specName)
```

Comment:
nit: no need to cat, grep can operate on the file.

Reply:
fixed